hw/mcu/dialog/da1469x/src/hal_flash.c

```diff
@@ -24,6 +24,7 @@
 #include "mcu/da1469x_hal.h"
 #include "hal/hal_flash_int.h"
 #include "mcu/mcu.h"
+#include <os/util.h>
```

Comment:
This probably should be moved to hal_bsp.c

Reply:
Done.  Will push refresh.

Reply:
Well that's embarrassing.  Moving the rdids from mcu to bsp, I forgot to update the other bsp.  We use our own external bsp code.  Fixed and pushed up.